internal/sqlutil/sql.go

```diff
@@ -107,3 +111,40 @@ func SQLiteDriverName() string {
 	}
 	return "sqlite3"
 }
+
+func minOfInts(a, b int) int {
+	if a <= b {
+		return a
+	}
+	return b
+}
+
+type QueryProvider interface {
+	QueryContext(ctx context.Context, query string, args ...interface{}) (*sql.Rows, error)
+}
+
+const SQLite3MaxVariables = 999
+
+//RunLimitedVariablesQuery split up a query with more varaibles than the used databse can handle in multiple queries.
```

Comment:
```suggestion
//RunLimitedVariablesQuery split up a query with more varaibles than the used database can handle in multiple queries.
```

Reply:
Will do. And also the varaibles thingy :-)